auth-connector/src/test/java/com/sequenceiq/cloudbreak/auth/security/authentication/UmsAuthenticationServiceTest.java

```diff
@@ -30,11 +30,23 @@ public void before() {
     }
 
     @Test(expected = UmsAuthenticationException.class)
-    public void testInvalidCrn() {
+    public void testInvalidCrnDueToPattern() {
+        String crn = "crsdfadsfdsf sadasf3-df81ae585e10";
         try {
-            underTest.getCloudbreakUser("crsdfadsfdsf sadasf3-df81ae585e10", "principal");
+            underTest.getCloudbreakUser(crn, "principal");
+        } catch (UmsAuthenticationException e) {
+            assertEquals("Invalid CRN has been provided: " + crn, e.getMessage());
+            throw e;
+        }
+    }
+
+    @Test(expected = UmsAuthenticationException.class)
```

Comment:
Please, avoid using Test(expected=), you can use this form:
https://github.com/hortonworks/cloudbreak/blob/4907b414595516775688efc32a9432cdbd7e4615/datalake/src/test/java/com/sequenceiq/datalake/service/sdx/ProvisionerServiceTest.java#L106

Or the old way:
https://github.com/hortonworks/cloudbreak/blob/4907b414595516775688efc32a9432cdbd7e4615/template-manager-core/src/test/java/com/sequenceiq/cloudbreak/template/filesystem/AbstractFileSystemConfiguratorTest.java#L25

Reply:
This is JUnit 4 code, so I'll switch to using ExpectedException. Thanks!